### DISCUSSION TITLE
Choose direffent movie version?

### DISCUSSION THREAD

**Discussion by chivigoku** [croneter/PlexKodiConnect#1977] [General]
Hi there,

I have some movies, in the same folder, with different versions. For example, movies in 4K Remux and RIP.
Is it possible to choose one of then when I want to play them?

I do not see that option anywhere.

Cheers?

**Comment by ThomasKat** [comment]
Use the Kodi Context menu ("c"), then PKC Settings, then choose "Play via PMS". It will ask you which version you want to play
